python/lsst/ip/isr/ptcDataset.py

```diff
@@ -242,10 +242,16 @@ def setAmpValues(self, ampName, inputExpIdPair=[(np.nan, np.nan)], expIdMask=[np
         self.ptcFitParsError[ampName] = ptcFitParsError
         self.ptcFitChiSq[ampName] = ptcFitChiSq
         self.ptcTurnoff[ampName] = ptcTurnoff
-        self.covariances[ampName] = covArray
-        self.covariancesSqrtWeights[ampName] = covSqrtWeights
-        self.covariancesModel[ampName] = covArrayModel
-        self.covariancesModelNoB[ampName] = covArrayModelNoB
+        if len(self.covariances[ampName]) == 0:
+            self.covariances[ampName] = np.array(covArray)
+        else:
+            self.covariances[ampName] = np.concatenate((self.covariances[ampName],np.array(covArray)))
```

Comment:
I'm concerned about dimensionality here.  Doesn't this take the `NxN` covariances and stack them to create an `(m_subregion*N)xN array?